vehicle/OVMS.V3/components/vehicle_renaultzoe/src/rz_web.cpp

```diff
@@ -131,6 +134,9 @@ void OvmsVehicleRenaultZoe::WebCfgFeatures(PageEntry_t& p, PageContext_t& c)
   c.input_select_option("Kangoo",    "1", vehicle_type == "1");
   c.input_select_end();
   
+  c.print("<hr>");
+  c.input_checkbox("use EGPIOs for Homelink", "use_gpios", use_gpios,
+    "<p>Disabeld are Default use for Homelink 1 = Climatecontrol for IOS users. When enable some EGPIOs are switched. (BINKINO thinks)</p>");
```

Comment:
This is an even worse explanation. No normal user will understand this, and software should generally not describe it's functionality by means of "some" and "thinks". It should exactly define what it does.